src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Link to the stylesheet should have a type attribute with the value 'text/css', and the href attribute should point to a .css file, not a .scss file. SCSS files are preprocessed and should not be linked directly in HTML.